ci-operator/config/coreos/bootupd/OWNERS

```diff
@@ -0,0 +1,14 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- ashcrow
+- cgwalters
+- jlebon
+- miabbott
+options: {}
+reviewers:
+- ashcrow
+- cgwalters
+- jlebon
+- miabbott
```

Comment:
What about expanding these lists to include @darkmuggle and @lucab?

Reply:
Sure, though I think we should do that as a second step; this is just copying existing files.

Reply:
OK looking around I found out we can use symlinks for `OWNERS`, so I did that for now.